certified-connectors/Maximizer/apiProperties.json

```diff
@@ -4,593 +4,32 @@
-                "tooltip": "Provide  Azure Active Directory credentials",
-                "constraints": {
-                  "required": "true",
-                  "hidden": "false"
-                }
-              }
-            },
-            "token:customHost": {
-              "type": "string",
-              "uiDefinition": {
-                "constraints": {
-                  "required": "true",
-                  "tabIndex": 3,
-                  "allowedValues": [
-                    {
-                      "text": "default",
-                      "value": "uk1.maximizercrmlive.com"
-                    }
-                  ]
-                },
-                "displayName": "Host",
-                "description": "Select default to continue",
-                "tooltip": "Save your notes about this connection."
-              }
-            },
-            "token:customBasePath": {
-              "type": "string",
-              "uiDefinition": {
-                "constraints": {
-                  "required": "true",
-                  "tabIndex": 3,
-                  "allowedValues": [
-                    {
-                      "text": "default",
-                      "value": "/intMSFlows"
-                    }
-                  ]
-                },
-                "displayName": "Integration path",
-                "description": "Select default to continue",
-                "tooltip": "Save your notes about this connection."
-              }
-            },
-            "token:customClientId": {
-              "type": "string",
-              "uiDefinition": {
-                "constraints": {
-                  "required": "true",
-                  "tabIndex": 3,
-                  "allowedValues": [
-                    {
-                      "text": "default",
-                      "value": "{{OAuth2UKcustomClientId}}"
-                    }
-                  ]
-                },
-                "description": "Select default to continue",
-                "displayName": "Client Id",
-                "tooltip": "Save your notes about this connection."
-              }
-            },
-            "token:customClientSecret": {
-              "type": "string",
-
-              "uiDefinition": {
-                "constraints": {
-                  "required": "true",
-                  "tabIndex": 3,
-                  "allowedValues": [
-                    {
-                      "text": "default",
-                      "value": "{{OAuth2UKcustomClientSecret}}"
-                    }
-                  ]
-                },
-                "description": "Select default to continue",
-                "displayName": "Client Secret",
-                "tooltip": "Save your notes about this connection."
-              }
-            }
-          }
-        }
-      ]
-    },
-    "iconBrandColor": "#c8ec25",
+    "iconBrandColor": "#007ee5",

```

Comment:
@davidchanmaximizer this Micorosft standard background color and it is not allowed to use it. can you please make sure this match you icon background color?

Reply:
@Amjed-Ayoub thank you for the remind, it is changed back to c8ec25 in the new commit.